modules/gatsby-theme-ocular/src/utils/links-utils.js

```diff
@@ -76,6 +82,18 @@ function addToRelativeLinks({source, target, rootFolder, edge, relativeLinks}) {
   };
 }
 
+function removeURLPath(relPath, pathToRemove) {
```

Comment:
Let's add some unit tests for this function.

Reply:
Done. Please check if this is enough.